Helpers/opent2t-translator-helper-hue/js/tests/README.md

```diff
@@ -0,0 +1,24 @@
+# Running Tests
```

Comment:
Both wink and nest helpers have the readme under /js/tests and did not have the readme for the helper itself. Should we keep hue helper consistent with wink and nest helpers?